src/Command/ContentHubAuditSslCertificate.php

```diff
@@ -47,14 +47,16 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     }
     // @TODO: This needs to be fixed. The 'https' is not present in the URL.
     if (strpos($url, 'https') !== 0) {
-      $output->writeln("<warning>This site does not have an SSL Certificate.</warning>");
+      $output->writeln("<warning>This site does not have an SSL Certificate or isn't publicly accessible.</warning>");
```

Comment:
I think this message **Site isn't publicly accessible** should be added inside catch because this condition just checks if site url contains https or not.